service/AppDynamicsService.go

```diff
@@ -48,6 +52,10 @@ func SendAppdynamicsAlert(message string) {
 		var buffer bytes.Buffer
 		buffer.WriteString(emoji.Sprintf(":red_circle:"))
 		buffer.WriteString(" ")
+		buffer.WriteString(businessMessage)
```

Comment:
Technical messages need to go to the technical group. Business Messages to the business group.